### ISSUE TITLE
TexLive Freeze Period 

### ISSUE THREAD

**Issue by d-alex-hughes** [rocker-org/rocker#519]
This is a question generated by the annual release of TexLive (hereafter TL). The spring of each year the TL developers freeze the last annual version and begin the release process for the new vintage. 

This year, their planned release timeline is the following (with some slight editing for date formatting)

> Plan for TeX Live 2023: 
> 
> Feb 10: candidate/final sources committed, test builds begin. 
> Feb 19: tlpretest starts, CTAN updates continue there and in tlnet. 
> Mar 3: code freeze for final build, major bug fixes only. 
> Mar 10: final updates from CTAN, final doc tweaks. 
> Mar 12: deliver TL image for TeX Collection packaging/testing. 
> Mar 19: public release (also of MacTeX). 
> May ?: delivery of TeX Collection DVD to members.

At the time that I'm documenting the issue we're in the middle of this release period. 

# The Background

As a consequence of the release, Rocker images that are built against 2022 versions TL now throw a warning, and engage in a time consuming build/re-build loop within the image. 

The warning that most users will notice is "TeX Live 2022 is frozen". I'll write here, for their benefit, that this is a warning only, and doesn't signify that there is a problem _per se_. Instead, it is just a notice to users that we're in the annual timeline listed above. The image _does_ eventually build (it is a ~3-5 minute process for the first knit in a session). 

<img width="954" alt="Screenshot 2023-03-13 at 6 52 10 PM" src="https://user-images.githubusercontent.com/6474241/224871782-c3373380-c608-4937-aa6e-0273552dac84.png">

I'll note that this is a problem that also manifests if someone runs the `rocker/tidyverse` without the TeX distribution, and then attempts to install TeX within a container using something sensible like `tinytex::install_tinytex()`. 

That is, this is really a tex release issue more than a rocker issue. 

# Question

I'm not sure where you'd like to file this issue (and feel free to close the issue without responding since the problem will resolve itself in a few weeks without intervention from the project): 

- Is there a way to design the rocker stack out of this recurring problem? 

On other R services that I contribute to (also at Berkeley, Carl!) we've built Ubuntu TeX on top of a rocker image (link)[https://github.com/berkeley-dsep-infra/datahub/blob/e9d48856af7498b2f0f2323b0e0d8afac4c267b7/deployments/ischool/image/Dockerfile#L40] which we can confirm is passing without warning now.

**Comment by eddelbuettel** [comment]
Which of the several rocker images are you seeing this with?  Possibly in the 'versioned2' stack?  I tend to look after base images and they do little to no latexing.

**Comment by d-alex-hughes** [comment]
Yes, I believe that they are coming from the `versioned2` stack.
